smart-contracts/contracts/mixins/MixinRoles.sol

```diff
@@ -51,11 +51,11 @@ contract MixinRoles is
     return hasRole(LOCK_MANAGER_ROLE, account);
   }
 
-  function addLockManager(address account) public {
-    _onlyLockManager();
-    grantRole(LOCK_MANAGER_ROLE, account);
-    emit LockManagerAdded(account);
-  }
+  // function addLockManager(address account) public {
+  //   _onlyLockManager();
+  //   grantRole(LOCK_MANAGER_ROLE, account);
+  //   emit LockManagerAdded(account);
+  // }
```

Comment:
as expected we are running into contract size issue... The removal of `addLockManager` function is just a dirty workaround that should NOT be merged